app/navigation/JourneyNavigator.scala

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package navigation
+
+import javax.inject.{Inject, Singleton}
+import play.api.mvc.Call
+import controllers.routes
+import pages._
+import models._
+import models.aboutyou.UkResidenceStatus
+import pages.aboutyou.{UkResidenceStatusPage, YourResidenceStatusPage}
+
+@Singleton
+class JourneyNavigator @Inject()() extends Navigator {
+  override def normalRoutes(taxYear: Int): Page => UserAnswers => Call = {
```

Comment:
Could we have not used the taxYear defined on userAnswers at this point?

Reply:
Yes we could, making the change.